app/livechat/client/views/app/tabbar/chatRoomSearchItem.js

```diff
@@ -0,0 +1,8 @@
+import moment from 'moment';
```

Comment:
You don't need this template, you can use the `chatRoomHistoryItem`.
Please, remove it.